packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -89,27 +87,36 @@ async function buildSwingset(
   return { controller, mb, bridgeInbound, timer };
 }
 
-function computronCounter(limit, vatCost) {
-  assert.typeof(limit, 'bigint');
-  assert.typeof(vatCost, 'bigint');
-  let total = 0n;
+function computronCounter({
```

Comment:
if we go with max-beans instead of max-computrons, this function would be called `beanCounter`, and the pun / etymological backstory would be complete

(but I'm still inclined to stick with max-computrons)